app/experimenter/experiments/api/v5/query.py

```diff
@@ -16,9 +16,15 @@
 )
 
 
+class ApplicationChannel(graphene.ObjectType):
+    label = graphene.String()
```

Comment:
`label` here could be `application` to be a little more readable?

Reply:
I called it label to match that its the label for the application in the list.